lib/matplotlib/backend_managers.py

```diff
@@ -14,6 +18,197 @@
 from matplotlib.rcsetup import validate_stringlist
 import matplotlib.backend_tools as tools
 
+from matplotlib import is_interactive
+from matplotlib import rcParams
+from matplotlib.figure import Figure
+from matplotlib.backends import get_backend, backend as backend_name
```

Comment:
Isn't this one of the sources of the global state that needs to be pushed back into `pyplot.py`